sdk/storage/Azure.Storage.Blobs.Cryptography/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
+===============
+
+## 12.0.0-preview.5
```

Comment:
Why are we putting versions that don't exist in our CHANGELOGs?  As a customer I'd find this very confusing.

Reply:
That's to get the tool passing initially. The idea is to always have the latest version present. so basically when the version is bumped up a new changelog entry to be started.

Reply:
I don't love that because as soon as we release and customers go to check what changed, there's going to be a version in this file that's different than what they can find on NuGet.

Reply:
Assuming they link to the file from Nuget it will be the version (SHA) at the time of release so they will be looking at the correct thing.

Reply:
Are you talking about the Project Site link on the side?  I don't think anybody's going to find our changelog by clicking that.  We don't link them in the README.md files so they'd have to navigate to the files view and start clicking around.

But there's a link to the latest copy right at the top of https://www.nuget.org/packages/Azure.Storage.Blobs/ which is very easy for users to find.

Reply:
@tg-msft, I solved that in Key Vault by using the tagged commit link to the changelog/readme that *will* exist once published. It uses a deterministic URL. Granted, that will make link validation more difficult (and I did point this out to Pavel).

Reply:
I suppose that could work, but it seems like a thing partner teams are very likely to get wrong when publishing on their own.

Reply:
@tg-msft You are right that does not happen right now. However my next PR after this will add CHANGELOG.md to the Nuget Package and also link to the release SHA version of the changelog.
[This EventHub package](https://www.nuget.org/packages/Azure.Messaging.EventHubs/5.0.0-preview.6) is an example of how it'll look like

Reply:
>@tg-msft Ted Glaza FTE, I solved that in Key Vault by using the tagged commit link to the changelog/readme that will exist once published. It uses a deterministic URL. Granted, that will make link validation more difficult (and I did point this out to Pavel).

As for links in the md files themselve the goal will be to use relative path links there and as part of the publishing process those relative links will be turned into absolute paths based on the sha which means they will always be stable.

Reply:
@weshaggard, cool, but to what I was referring was that I used such a link in the release notes of the nupkg, but it looks like that will be solved by @chidozieononiwu adding CHANGELOG.md to the package and including a link to the commitish.

Reply:
It would be super helpful if we could share these plans before sending PRs because it's hard to evaluate changes like this without context.  It would also be good to share what we think the final NuGet format is going to look like with our partner teams now because some of them have strong opinions about what's included there.